app/views/layouts/application.html.erb

```diff
@@ -57,7 +57,7 @@
   <main id="content">
    
   <div class="phase-banner phase-banner-beta">
-    <%= t('.feedback_banner_html', url: new_feedback_submission_path) %>
+    <%= t('.feedback_banner_html', url: 'https://visit-someone-in-prison.form.service.justice.gov.uk') %>
```

Comment:
`new_feedback_submission_path` was actually previously used to go to a page that had a submittable feedback form, this has previously been removed (I can't find where/when) and now simply serves a static information page. I've decided instead of trying to change old unused code to work for our new purpose, it would be better to no longer send users to this page.
This means that if/when we go to remove the feedback route entirely, it will be tied in to a lot less code so should be easier to remove correctly (since it is effectively a disable feature no longer serving it intended purpose)